multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -435,7 +432,7 @@ fn expire_payables(path: PathBuf) {
     statement.execute([]).unwrap();
 
     let mut config_stmt = conn
-        .prepare("update config set value = '0' where name = 'start_block'")
+        .prepare("update config set value = '0' where name = 'start_block' and (value is null or value < 1)")
```

Comment:
I'll revert it. It was a fleeting thought to use the same thing in production code to prevent undoing progress by storing a smaller number for the config value. Then I realized we don't have the ability to have a custom where clause to it.